backend/substrapp/compute_tasks/outputs.py

```diff
@@ -85,3 +137,14 @@ def _get_perf(perf_path: str) -> float:
     with open(perf_path, "r") as perf_file:
         perf = json.load(perf_file)["all"]
         return float(perf)
+
+
+def _get_model_category(identifier: str) -> str:
+    """Temporarily hardcode a match between model identifier and category.
+
+    This is based on the following assumptions and should be removed once we drop model categories:
+    - all models are SIMPLE
+    - except "local" or "head" outputs
+    """
+
+    return model_pb2.MODEL_HEAD if identifier.lower() in ["local", "head"] else model_pb2.MODEL_SIMPLE
```

Comment:
Can you avoid added hardcoded literals in the function body? e.g. by using top-level constants

```py
OUTPUT_IDENTIFIER_LOCAL = "local"
```

or by refactoring with 

https://github.com/Substra/substra-backend/blob/a780df98e907376378607e4bcb5f8d286c9f5507/backend/localrep/serializers/computetask.py#L36-L41

Reply:
The refactor you propose is not behaving the same way, I extracted the identifiers in a constant.